DependencyInjection/DoctrineExtension.php

```diff
@@ -91,6 +92,21 @@ public function load(array $configs, ContainerBuilder $container)
         $config        = $this->processConfiguration($configuration, $configs);
 
         if (! empty($config['dbal'])) {
+            // if no DB connection defined, prepend an empty one for the default
```

Comment:
good question...
I would say that it does not matter given it only adds an empty connection array and that there's no change it would get overridden because otherwise, we would not be in that case.
but my gut tells me pre-pending is better because it's closer to the concept of `processConfiguration`

Reply:
but now that I read your comment bellow your question makes more sense to me...
test passes so I guess we are good to go